diesel/src/query_source/mod.rs

```diff
@@ -46,26 +46,31 @@ pub trait QuerySource {
 /// been generated by the [`table!` macro](crate::table!).
 pub trait Column: Expression {
     /// The table which this column belongs to
-    type Table: Table;
+    type Source: QuerySource;
```

Comment:
While I can understand the motivation for renaming this field, it remains a breaking change. I would rather go with
```suggestion
    type Table: Releation;
```
instead. (As `Relation` is the super trait of `Table` that's fine)

Maybe we can also add a `Fixme(diesel 3.0): rename to Source` or something like that.